### ISSUE TITLE
Unable to pass arguments when launching ZAP from the command line on Mac OS X

### ISSUE THREAD

**Issue by davehunt** [zaproxy/zaproxy#1715]
Command line arguments are being ignored when launching ZAP from the terminal on Mac OS X.

I'm expecting the following to launch ZAP in daemon mode:
```shell
open -a "/Applications/OWASP ZAP.app" --args --daemon
```
The following works as I'd expect:
```shell
bash /Applications/OWASP\ ZAP.app/Contents/Java/zap.sh -daemon
```

**Comment by psiinon** [comment]
This is currently the top issue on https://www.bountysource.com/teams/zap/issues (with 2 votes;)

**Comment by marumari** [comment]
This should be fixed in #1728.  Please let me know if everything works for you.  :)

**Comment by psiinon** [comment]
That did it :D
